util/app-config/src/configs/store.rs

```diff
@@ -40,7 +38,6 @@ impl Default for Config {
             block_tx_hashes_cache_size: 30,
             block_uncles_cache_size: 30,
             block_extensions_cache_size: default_block_extensions_cache_size(),
-            cellbase_cache_size: 30,
             freezer_enable: false,
```

Comment:
This config can be ignored, but can't delete directly, it hurt backward compatibility.